create_amazon_bucket.md

```diff
@@ -0,0 +1,76 @@
+## Create a new amazon bucket
+
+### Bucket
+We use the naming convention `<product-name>-production` and `<product-name>-staging`
+
+
+#### Region
+We normally default to use `eu-north-1` aka. `Stockholm` but this may vary depending on where you and/or your customers are, so if another region makes more sense choose a different one.
+
+#### Settings
+Under the section `Configure options` we disable everything as a default, it's only if the project requires something specific such as versioning or server logging that you should enable it.
+
+The only thing that is recommended, is to add the `default encryption` with AES-256. This should be used if the bucket is going to handle sensitive information (ID's, documentation that can be referenced back to the client etc.).
+
+#### Permissions
+Block all public access, unless you require otherwise.
+It's fine to block all public access, you just have to create a token in your service that grants access to the data (Remember to set a time limit on the token).
+
+### IAM user
+No app should ever log in as the root user! Nor the login that was used to create the buckets. Create a IAM account for each bucket created, that way if one account is compromised potential damage is kept to a minimum and easy to fix.
+
+1. Goto Security, Identity & Compliance < IAM
+2. Then Access Management < Users
+3. Click add user.
+
+
+In order to easily identify which user connects to which bucket, name them both the same.  Since this is a user only used through the service/application, the only `access type` it requires is `Programmatic access`.
+
+
+Nothing else needs to be added to the user, just click 'Next' until you get to a page showing you the `Access Key ID` and the `Secret Access Key`. These keys need to be stored in a secure location. Add them directly to the `.env` file, store them in a password manager, so the team also has easy access to the information, or both.
```

Comment:
Only the staging/development users' keys should be added to `.env.development`. It shouldn't be done with the production ones.

Reply:
```suggestion
Nothing else needs to be added to the user, just click 'Next' until you get to a page showing you the `Access Key ID` and the `Secret Access Key`. These keys need to be stored in a secure location. For the development user/bucket, add the keys directly to the `.env.development` file, store them in a password manager, so the team also has easy access to the information, or both.
```

Reply:
Good catch, will change the text